CRM/Mailing/BAO/Mailing.php

```diff
@@ -1731,7 +1731,6 @@ public static function create(&$params, $ids = array()) {
     CRM_Contact_BAO_Contact_Utils::generateChecksum($mailing->id, NULL, NULL, NULL, 'mailing', 16);
 
     $groupTableName = CRM_Contact_BAO_Group::getTableName();
-    $mailingTableName = CRM_Mailing_BAO_Mailing::getTableName();
```

Comment:
(note: this is unrelated to the subject of the PR, but `$mailingTableName` goes unused so this is a good bit of code cleanup.)